test-extra/Makefile

```diff
@@ -53,8 +57,21 @@ code/dune: URI = https://github.com/ocaml/dune.git
 # code/alcotest: URI = https://github.com/mirage/alcotest
 # code/citty: URI = https://github.com/ocurrent/citty
 
+# include similar rules from the janestreet repositories
+-include js.mak
+
+js-repo:
+	@git clone https://github.com/janestreet/opam-repository.git js-repo
```

Comment:
This is too big to be practical. I'd prefer if there were a few important projects rather than all of them.